Dockerfile

```diff
@@ -21,23 +21,29 @@
 #           -e INPUT_REPO_TOKEN=$GITHUB_AUTH_TOKEN \
```

Comment:
Let's not update the Dockerfile in this PR. It will roll out Golang based scorecard-action to end users (whenever we create a new release or for anyone brave enough to be using `@main` ref).